index.html

```diff
@@ -49,9 +72,214 @@ <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
 
 
 
+
+
+
+
+    <div class="main">
+    
+        <div class="Works">
+          <div class="img-1"><img class="fontimg"  width="100%"  height="100%" src="Snapshoot1.png" alt=""></div>
+          
+          <div class="tonic">
+              <div><h2>Tonic</h2></div>
+              <div class="flex1">
+                  <div>Caponi</div>
+                  <div class="round"></div>
+                  <div>back End</div>
+                  <div class="round"></div>
+                  <div>2015</div>
+              </div>
+              <div>
+                  <p>
+                      A dayly selection of privatelypersonalized reads:no accounts <br>
+                      or sign-ups required.
+                  </p>
+              </div>
+              <div >
+                  <ul class="languages">
+                  <li class="language">html</li>
+                  <li class="language">css</li>
+                  <li class="language">javascript</li>
+                  </ul>
+              </div>
+              <div><button class="see-project" type="button">See project</button></div>
+          </div>
+            </div>
+          
+      
+      
+      
+            <div class="Works">
```

Comment:
- [ ] [**OPTIONAL**] Kindly keep doing the  great work by remove this bank lines 👍  it  is crucial to clean readable code